### DISCUSSION TITLE
Front page view

### DISCUSSION THREAD

**Discussion by piraterx** [stashapp/stash#2621] [Ideas]
I liked old way where stats are viewed and not content as if we accidentally click on link from start page, there isn't anything to see. Is there a way to give option what is visible on that page and what not or old way where we just see stats.

**Comment by ALonelyJuicebox** [comment]
Counter argument-- if you're accessing Stash from a bookmark as it is, why not change your bookmark to navigate you to the Stats page? 🤷🏾‍♂️

**Reply by piraterx** [reply]
I think that make sense as well.

**Comment by cj12312021** [comment]
@piraterx I'm not how possible this would be as Stash continues to evolve. The stats page is still visible on the navbar to view, and as JuiceBox mentioned, you could bookmark that page instead. Now that the stats page no longer has to serve as the front page, there has been talk about transforming it into a proper statistic page. It would be even less suitable on the front page if that happens.

**Reply by piraterx** [reply]
Got it. As long as static page doesn't have any nude image, it should be good. I will use that as bookmark.

**Comment by cj12312021** [comment]
We could consider supporting widgets on the front page that users could use to bring the content counter back. This is likely in line with the vision @WithoutPants had for the front page configuration page.
